### PR TITLE
Landing page dashboard

### PR DESCRIPTION
**Pull request by vlilja**
Some charts for front-end, also fixed some bugs etc.

### CODE REVIEW

File: src/components/Dashboard/ChartBoard.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <loading-spinner v-if="!appInitialized" />
+    <div v-else>
+      <div v-if="skillList.length === 0 || skillProfiles.length === 0">
+        <span id="no-data-message">Sorry there is no data to display :(</span>
+      </div>
+      <div v-else>
+        <b-row class="no-gutters mt-1">
+          <top-skill-chart
+            :skill-info="skillInfo"
+            :size="5" />
```
Comment (LKajan) [inline src/components/Dashboard/ChartBoard.vue]: cols=12 md=6 could work better also on smaller screens.

File: src/components/Dashboard/SkillChart.vue [hunk 2]
```diff
@@ -0,0 +1,209 @@
+<template>
+  <chart-card
+    :size="size">
+    <template slot="header">
+      <div class="h3">Skills <i
```
Comment (LKajan) [inline src/components/Dashboard/SkillChart.vue]: Vertical alignment of text and dropdown box doesn't match. Looks little bit messy.

File: src/components/Dashboard/ChartBoard.vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      Object.keys(skillProfienciesByGroup).forEach((skillId) => {
+        const skill = skillProfienciesByGroup[skillId]
+        skill.proficiencyAverage = this.calculateAverage(skill.proficiencies)
+        skill.willingnessAverage = this.calculateAverage(skill.willingness)
+      })
+      return skillProfienciesByGroup
+    }
+  },
+  methods: {
+    calculateAverage (arr) {
```
Comment (LKajan) [inline src/components/Dashboard/ChartBoard.vue]: Should these general functions that don't need access to vue instance be moved out of methods?

File: src/views/LandingPage.vue [hunk 4]
```diff
@@ -18,14 +18,19 @@
         Ready to Rock ’n’ Roll
       </template>
     </b-jumbotron>
+    <dashboard v-if="isAuthenticated"/>
```
Comment (LKajan) [inline src/views/LandingPage.vue]: I would replace the whole landing page with this.

File: src/components/Dashboard/ChartBoard.vue [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+            class="pb-1"/>
+        </b-row>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import LoadingSpinner from '../helpers/LoadingSpinner'
```
[the remark this answers is not in the document]
Comment (vlilja) [reply src/components/Dashboard/ChartBoard.vue]: Changed to use global component and moved the global component from lib -> helpers

File: src/components/Dashboard/SkillChart.vue (hunk 2 above)
Comment (vlilja) [reply src/components/Dashboard/SkillChart.vue]: This is cause of one not scoped style block that is fixed in another pull request

File: src/components/Dashboard/ChartBoard.vue (hunk 3 above)
Comment (vlilja) [reply src/components/Dashboard/ChartBoard.vue]: moved from component methods to class scope

**Pull request merged**